### ISSUE TITLE
Import package issue

### ISSUE THREAD

**Issue by gaow** [bpeng2000/SOS#71]
I'm using SoS for another project where I need sklearn package, e.g. on conda: `conda install sklearn`. If I import the package in Python it works well (no error reported):

```
python -c "from sklearn import manifold"
```

But if I use the following SoS script
```
from sklearn import manifold

[run]
print(0)
```
It quits on  `AttributeError: 'Process' object has no attribute '_config'`. The complete trace is below, which looks like an issue with sklearn, but I'm not sure why SoS cannot load when Python works fine ...

```

  File "/home/shared/miniconda3/lib/python3.5/site-packages/sos-0.5.6-py3.5.egg/pysos/sos_script.py", line 772, in run
    SoS_exec(self.global_process)
Traceback (most recent call last):
  File "/home/shared/miniconda3/lib/python3.5/site-packages/sos-0.5.6-py3.5.egg/pysos/sos_script.py", line 772, in run
    SoS_exec(self.global_process)
  File "/home/shared/miniconda3/lib/python3.5/site-packages/sos-0.5.6-py3.5.egg/pysos/utils.py", line 785, in SoS_exec
    exec(stmts, env.sos_dict._dict)
  File "<string>", line 1, in <module>
  File "/home/shared/miniconda3/lib/python3.5/site-packages/sklearn/__init__.py", line 57, in <module>
    from .base import clone
  File "/home/shared/miniconda3/lib/python3.5/site-packages/sklearn/base.py", line 11, in <module>
    from .utils.fixes import signature
AttributeError: 'Process' object has no attribute '_config'
  File "/home/shared/miniconda3/lib/python3.5/site-packages/sos-0.5.6-py3.5.egg/pysos/sos_script.py", line 1717, in sos_run
    workflow.run(workflow_args, cmd_name='{} {}'.format(args.script, args.workflow), config_file=args.__config__)
Traceback (most recent call last):
  File "/home/shared/miniconda3/lib/python3.5/site-packages/sos-0.5.6-py3.5.egg/pysos/sos_script.py", line 1717, in sos_run
    workflow.run(workflow_args, cmd_name='{} {}'.format(args.script, args.workflow), config_file=args.__config__)
  File "/home/shared/miniconda3/lib/python3.5/site-packages/sos-0.5.6-py3.5.egg/pysos/sos_script.py", line 1181, in run
    raise RuntimeError(res)
RuntimeError: Failed to execute statements
"from sklearn import manifold

"
'Process' object has no attribute '_config'
```

**Comment by bpeng2000** [comment]
```
$ conda install sklearn
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata: ....
Solving package specifications: .
Error:  Package missing in current osx-64 channels: 
  - sklearn

Did you mean one of these?

    scikit-learn, seaborn

You can search for this package on anaconda.org with

    anaconda search -t conda sklearn
```

**Comment by gaow** [comment]
Sorry I meant to use `scikit-learn`

**Comment by gaow** [comment]
Actually sorry I installed via `pip install sklearn`

**Comment by bpeng2000** [comment]
Still checking, but

```
    from .utils.fixes import signature
```

looks scary, especially if `signature` is imported from `inspect`.

**Comment by bpeng2000** [comment]
http://stackoverflow.com/questions/27904162/using-multiprocessing-pool-from-celery-task-raises-exception

**Comment by gaow** [comment]
Oh, ouch ... sounds like we need that manual fix.

**Comment by bpeng2000** [comment]
The strangest thing is that if I switch back to multiprocessing under Python 3.5.1, the `testPassingVarsToNestedWorkflow` works. Perhaps python has done something to the `multiprocessing` module? If this is confirmed, I have no problem looking back and check which version of python fixed the problem and requires it.

I have submitted a patch to switch back to `multiprocessing`. I have also added a test for this case so that I can check around and see which test would fail under which condition etc.
